os/arch/arm/src/armv8-m/up_assert.c

```diff
@@ -306,12 +306,12 @@ static void check_sp_corruption(uint32_t sp, uint32_t *stackbase, uint32_t *stac
 {
 	lldbg_noarg("===========================================================\n");
 	lldbg_noarg("Asserted task's stack details\n");
-	lldbg_noarg("===========================================================\n");	
+	lldbg_noarg("===========================================================\n");
```

Comment:
Please revert this extra space at the end

Reply:
same as above